configs/components/fesom/fesom-2.6.yaml

```diff
@@ -24,13 +24,11 @@ required_plugins:
 
 choose_version:
   "2.6":
-    branch: "2.6.7"
+    branch: "2.6.6"
 
-  "2.6-paleodyn":
+  "2.6-7":
     branch: "2.6.7"
-    wiso_code: true
-    icb_code: true
-    namelist_dir: "${model_dir}/config/"
```

Comment:
Why does these lines need to be removed?

Reply:
This should not be part of a fesom version. These are just switches can be set for any fesom version.